configurable-rules/json-schema-misconfigurations/README.md

```diff
@@ -0,0 +1,116 @@
+# JSON Schema misconfigurations
+
+Authors:
+- [`adamaltman`](https://github.com/adamaltman) Adam Altman (Redocly)
+ 
+## What this does and why
+
+This catches common misconfigurations of JSON Schema:
+- defining minimum or maximum values for a string (these are more logical for a number)
+- defining items on an object (instead of an array)
+- defining properties on an array (instead of items)
+
+## Code
+
+The first rule checks that a string isn't using the `minimum` and `maximum` keywords.
+```yaml
+  rule/json-schema-string-misconfiguration:
+    subject: 
+      type: Schema
+      property: 
+        - minimum
+        - maximum
+    where: 
+      - subject: 
+          type: Schema
+          property: type
+        assertions:  
+          const: string
+    assertions: 
+      defined: false
```

Comment:
This would highlight the keywords instead of their values (which I think is more obvious): 

```suggestion
    subject: 
      type: Schema
    where: 
      - subject: 
          type: Schema
          property: type
        assertions:  
          const: string
    assertions: 
      disallowed: 
        - minimum
        - maximum
```

The same works for the other rules.

Reply:
Yes, I see what you mean here:

![image](https://github.com/Redocly/redocly-cli-cookbook/assets/1161871/6e49ca9d-f05e-479d-92ef-6ab46f16873c)

![image](https://github.com/Redocly/redocly-cli-cookbook/assets/1161871/2a48d156-3b25-4c89-9941-e256bbfab3da)